### PR TITLE
Feature/workout class

### PR DESCRIPTION
**Pull request by emilsavino**

### CODE REVIEW

File: src/test/java/WorkoutClassTest.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    @Test
+    public void testAddExercise() {
+        Exercise pushup = new Exercise("Pushup", "Push your body up and down", "Strength", "Chest", "None", "Easy");
+        testWorkout.addExercise(pushup);
+        Assert.assertEquals(1, testWorkout.getExercises().size());
+        Assert.assertEquals("Pushup", testWorkout.getExercises().get(0).title);
+        testWorkout.addExercise(new Exercise());
+        Assert.assertNotEquals(1, testWorkout.getExercises().size());
+        testWorkout.addExercise(new Exercise());
+        Assert.assertEquals(3, testWorkout.getExercises().size());
```
[the remark this answers is not in the document]
Comment (emilsavino) [reply src/test/java/WorkoutClassTest.java:29]: It does check if pushup is added on line 25

**Review by ThePhoneBeLinging — CHANGES_REQUESTED** [review]

File: src/main/java/Group15/Workout.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    public Workout() {}
+
+    public void addExercise(Exercise exercise) {
+        exercises.add(exercise);
+    }
+
+    public void swapExercise(Exercise toRemove, Exercise toAdd) {
+        if (exercises.contains(toRemove)) {
+            exercises.remove(toRemove);
+            exercises.add(toAdd);
```
[the remark this answers is not in the document]
Comment (ThePhoneBeLinging) [reply src/main/java/Group15/Workout.java:20]: And use the other specified methods, such as using the addExercise()

**Review by ThePhoneBeLinging — CHANGES_REQUESTED** [review]

**Review by ThePhoneBeLinging — APPROVED** [review]

File: src/main/java/Group15/Workout.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+public class Workout {
+    private List<Exercise> exercises = new ArrayList<>();
+
+    public Workout() {}
+
+    public void addExercise(Exercise exercise) {
+        exercises.add(exercise);
+    }
+
+    public void swapExercise(Exercise toRemove, Exercise toAdd) {
```
Comment (emilsavino) [inline src/main/java/Group15/Workout.java:17]: You forgot to return inside the if you stupid idiot

Comment (ThePhoneBeLinging) [reply src/main/java/Group15/Workout.java:17]: :)

Comment (ThePhoneBeLinging) [reply src/main/java/Group15/Workout.java:17]: How did that go through review, wtf :)

**Review by WettyWednesday — APPROVED** [review]

**Pull request merged**